Applications/Games/Dreamscapes/Online/script.json

```diff
@@ -0,0 +1,11 @@
+{
+    "scriptName"                 : "Online",
+    "id"                         : "applications.games.dreamscapes",
+    "compatibleOperatingSystems" : [
+        "MACOSX",
+        "LINUX"
+    ],
+    "testingOperatingSystems" : [],
+    "free"                    : false,
```

Comment:
The URL is `free-games-download`: Is it free or not?

Reply:
It is free of costs.
But it is not open source.
I will change it to true.
But I also suggest to distinguish between "free of cost" and free (regarding license).
So adding an additional field in future would be helpful, imho.

Reply:
Feel free to open a new issue to discuss this. Currently, it's free as in beer (see [docs](https://phoenicisorg.github.io/scripts/Develop/script-json/)).